### ISSUE TITLE
PDF export for RAF submission

### ISSUE THREAD

**Issue by melish** [eaudeweb/ozone#990]
plus corresponding button in the form
plus corresponding /reports, which allows multiple parties and periods and only shows current RAF submissions (not in data entry, not superseded, not recalled etc.)

**Comment by mihai-macaneata** [comment]
0e0f61d126b152e8fae0b0456863bc5208622428

**Comment by melish** [comment]
Done, except for second table (critical uses by category, which depends on #1110).
Will open a new ticket for the PDF
